frontend/src/component/admin/traffic/Traffic.tsx

```diff
@@ -0,0 +1,11 @@
+import InstanceMetricsChart from '../instance-admin/InstanceMetrics/InstanceMetricsChart'; // TODO move this to traffic folder (pending conversation on how to organize this)
```

Comment:
@chriswk maybe it's worth moving `InstanceMetricsChart` to frontend/src/component/admin/traffic and perhaps renaming it. I'm leaving this as a stylish comment.